web/src/routes/(user)/utilities/merge-faces/[personId]/+page.svelte

```diff
@@ -0,0 +1,181 @@
+  import { getPeopleThumbnailUrl } from '$lib/utils';
+  import { AppRoute } from '$lib/constants';
+  import UserPageLayout from '$lib/components/layouts/user-page-layout.svelte';
+  import { t } from 'svelte-i18n';
+  import Button from '$lib/components/elements/buttons/button.svelte';
+  import { mergePerson } from '@immich/sdk';
+  import {
+    notificationController,
+    NotificationType,
+  } from '$lib/components/shared-components/notification/notification';
+  import { goto } from '$app/navigation';
+  import Icon from '$lib/components/elements/icon.svelte';
+  import { mdiCheckCircle } from '@mdi/js';
+  import { SvelteSet } from 'svelte/reactivity';
+
+  let selectedPeople = new SvelteSet<string>();
+
+  interface Props {
+    data: PageData;
+  }
+  let { data }: Props = $props();
+
+  const clearSelection = () => {
+    selectedPeople.clear();
+  };
+
+  const handleKeydown = (e: KeyboardEvent) => {
+    if (e.key === 'Escape') {
+      clearSelection();
+    }
+  };
+
+  const toggleSelection = (personId: string) => {
+    if (selectedPeople.has(personId)) {
+      selectedPeople.delete(personId);
+    } else {
+      selectedPeople.add(personId);
+    }
+  };
+
+  const handleMerge = async () => {
+    if (selectedPeople.size === 0) {
+      return;
+    }
+
+    try {
+      await mergePerson({
+        id: $page.params.personId,
+        mergePersonDto: { ids: Array.from(selectedPeople) },
+      });
+
+      notificationController.show({
+        message: $t('merge_people_utility_success', {
+          values: {
+            count: selectedPeople.size,
+            name: data.person.name,
+          },
+        }),
+        type: NotificationType.Info,
+      });
+
+      await goto(AppRoute.MERGE_FACES);
+    } catch {
+      notificationController.show({
+        message: $t('errors.unable_to_merge_people'),
+        type: NotificationType.Error,
+      });
+    }
+  };
+
+  onMount(async () => {
+    const { people: similarPeople } = await getAllPeople({
+      withHidden: false,
+      closestPersonId: data.person.id,
+    });
+    // Filter out the person we searched for since we don't want to merge with themselves
+    const selfIndex = similarPeople.findIndex((person) => person.id === data.person.id);
+    if (selfIndex !== -1) {
+      similarPeople.splice(selfIndex, 1);
+    }
+    people = similarPeople;
+  });
+
+  let people: PersonResponseDto[] = $state([]);
+</script>
+
+<svelte:window onkeydown={handleKeydown} />
+
+<UserPageLayout title={$t('choose_faces_to_merge', { values: { name: data.person.name } })}>
+  {#snippet buttons()}
+    <div class="flex justify-end px-4">
+      <Button disabled={selectedPeople.size === 0} onclick={handleMerge} size="sm">
+        {$t('merge_selected', { values: { count: selectedPeople.size, person: data.person.name } })}
+      </Button>
+    </div>
+  {/snippet}
+  <div class="flex flex-col gap-4">
```

Comment:
I think this div doesn't accomplish anything, it looks like a flex container but it only has one child